### PR TITLE
add: Reloader deployment to osc-cl2 cluster

### PR DESCRIPTION
**Pull request by dystewart**
In response to #1778 adding reloader deployment to osc-cl2 cluster to track trino pods.

### CODE REVIEW

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm

File: cluster-scope/overlays/prod/osc/osc-cl2/kustomization.yaml [hunk 1]
```diff
@@ -13,13 +13,16 @@ resources:
...
+  - ../../../../base/core/namespaces/reloader
   - ../../../../base/operators.coreos.com/operatorgroups/openshift-nfd
   - ../../../../base/operators.coreos.com/subscriptions/nfd
   - ../../../../base/operators.coreos.com/subscriptions/opendatahub-operator
   - ../../../../base/config.openshift.io/oauths/cluster
   - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/acme-operator
   - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/cluster-admins-rb
+  - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/reloader-reloader-role-binding
   - ../../../../base/rbac.authorization.k8s.io/clusterroles/acme-operator
+  - ../../../../base/rbac.authorization.k8s.io/clusterroles/reloader-reloader-role
```
Comment (HumairAK) [inline cluster-scope/overlays/prod/osc/osc-cl2/kustomization.yaml:25]: include the bundle instead: https://github.com/operate-first/apps/tree/master/cluster-scope/bundles/reloader

example: 
https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L130

**Comment by HumairAK** [comment]
Can you also update the docs: https://www.operate-first.cloud/apps/content/reloader/monitoring-secrets-configmaps.html 

To mention that this is available in `smaug` and `osc-cl2`, maybe list them out

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**